nautobot_data_validation_engine/custom_validators.py

```diff
@@ -295,7 +298,7 @@ def compliance_result(self, message, attribute=None, valid=True):
             defaults={
                 "last_validation_date": self.result_date,
                 "validated_object_str": str(instance),
-                "validated_attribute_value": str(attribute_value) if attribute_value else "",
+                "validated_attribute_value": str(attribute_value)[:254] if attribute_value else "",
```

Comment:
Good catch! We should probably (at least in core/3.0) change this from a `CharField` to a `TextField` for more capability/flexibility.